File: src/BreakfastProvider.Api/Models/Responses/CustomerPreferenceResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public class CustomerPreferenceResponse
{
    public string CustomerId { get; set; } = string.Empty;
    public string CustomerName { get; set; } = string.Empty;
    public string PreferredMilkType { get; set; } = string.Empty;
    public bool LikesExtraToppings { get; set; }
    public string FavouriteItem { get; set; } = string.Empty;
    public DateTime UpdatedAt { get; set; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/CustomerPreferenceResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;

/** Twin of C# {@code CustomerPreferenceResponse}. */
public record CustomerPreferenceResponse(
        String customerId,
        String customerName,
        String preferredMilkType,
        boolean likesExtraToppings,
        String favouriteItem,
        Instant updatedAt) {
}
